test/e2eTests/rest/openMrsApis.js

```diff
@@ -1,51 +1,111 @@
 const assert = require('assert')
 
+const APIMANAGER_LOGTAG = '[ApiManager]'
```

Comment:
What do you think of this pattern. I like logs to show the class that produced them. It makes it easier to read logs and pinpoint issues. I've used this practice for a while and I've seen others use it, but there might be something better.

Reply:
I think it's a good pattern, should be helpful when debugging issues. I think we should use this is the app also.